### PR TITLE
Refactor UI

### LINKED ISSUE #1: Add webcam selector

### LINKED ISSUE #5: Show state in UI

### LINKED ISSUE #9: Refactor UI

### LINKED ISSUE #14: Add frontend node on keyboard input mode status
A small note in the frontend when keyboard input mode is enabled and whether or not the Keys are currently locked.

### PR DESCRIPTION
**Pull request by Importantus**
I refactored the UI by implementing a global pinia store and moving the facetracking into its own component. I also rewrote parts of the facetracking to be more bug resistant and to allow camera switching mid-tracking. Speaking of camera switching, I implemented that as well (there is still a small problem with showing the selected camera after a restart). 

I also started to make the UI look nicer ✨ and to show the current state of the board. The latter is finished, but I think the UI itself needs two or three more tweaks (especially the camera switcher and the button next to it).

Last thing is the debug badge I added. It is displayed when keyboard keys are being registered as debug options.

This PR should close issues #1 #5 #9 #14 and begin issue #6

### CODE REVIEW

**Review by KyRuS-SH — COMMENTED** [review]
Pretty good! Just a few adjustments needed:

File: src/components/NotConnected.vue
Comment (KyRuS-SH) [inline src/components/NotConnected.vue:1]: How do we correctly name the product in branding?
Dot Pad? dotpad? dot.pad?

File: src/App.vue
Comment (KyRuS-SH) [inline src/App.vue:1]: A status of the current Keylock would also be useful. However, we can move that to a different issue for now as it's not a priority.

File: electron/controller.ts
Comment (KyRuS-SH) [inline electron/controller.ts:1]: When testing _**Browser Control**_ couldn't be toggled off after enabling it on keyboard (presumably also not on dot pad?). The only way to get out of the mode is to go to a different mode.

Comment (KyRuS-SH) [reply electron/controller.ts:1]: Now that I'm looking at it, this could also be addressed in #2.

File: src/components/NotConnected.vue
Comment (Importantus) [reply src/components/NotConnected.vue:1]: I think in the logo that is printed on the controller, it is "dotpad". But I would say this is an issue for #8

File: electron/controller.ts
Comment (Importantus) [reply electron/controller.ts:1]: Yes. It's just not implemented in the controller yet. I will do that in #2 :)

**Pull request merged**